mypy/main.py

```diff
@@ -404,6 +404,7 @@ def add_invertible_flag(flag: str,
                         dest='special-opts:no_fast_parser',
                         help=argparse.SUPPRESS)
     if server_options:
+        # TODO: This flag is superfluous and should be removed after a short transition
```

Comment:
Add current date to the comment so that if somebody sees this comment a year from now they will immediately know that the flag can be removed :-)